packages/calypso-color-schemes/src/shared/color-schemes/_modern.scss

```diff
@@ -178,6 +179,7 @@ Uses custom theme highlight monochromatic palette for primary + accent
 	--color-navredesign-sidebar-menu-selected-text: var(--theme-text-color);
 	--color-navredesign-sidebar-submenu-selected-text: var(--theme-text-color);
 	--color-navredesign-sidebar-submenu-hover-text: #33f078; /* $menu-submenu-focus-text */
+	--color-navredesign-sidebar-submenu-selected-hover-text: var(--color-sidebar-submenu-hover-text);
```

Comment:
The distinction between `--color-sidebar-submenu-selected-hover-text` and `--color-navredesign-sidebar-submenu-selected-hover-text` is a little confusing.

I looked up the code history and found that the `Nav Redesign` variables were introduced in #90193. It's a little difficult to tell if adding those new variables was necessary or convenient… Knowing that might help inform us if we should add a single variable, or if adding both makes more sense.

Because all the CSS vars in the L632-L655 block in `client/my-sites/sidebar/style.scss` use the `--color-navredesign` prefix, I can see why you'd want to stick to that convention, @chad1008, but when there's no difference between `--color-sidebar-submenu-selected-hover-text` and `--color-navredesign-sidebar-submenu-selected-hover-text`, that does add some confusion.

@chad1008, what do you think?

Reply:
cc @nightnei here as well in case you have any insight from reviewing the other PRs

Reply:
>It's a little difficult to tell if adding those new variables was necessary or convenient

I had the same thoughts

>but when there's no difference between --color-sidebar-submenu-selected-hover-text and --color-navredesign-sidebar-submenu-selected-hover-text, that does add some confusion.

I had the same concern, but after thinking about it more - I made a decision that it's indeed very good idea to use an extra variable. I will try to express my thoughts: as you mentioned above - there are two different prefixes to distinguish styles, and if we reuse some variable - we would start mixing them, what is not good in case if we do some clean up or refactoring. Also it's potential issue for regression. I don't know, whether I managed to be clear... 😅

Reply:
@okmttdhr 👋 We're looking for some background on why you added new CSS variables with the `--color-navredesign` prefix in #90193. We're wondering whether we should continue adding more variables like that, and, if so, how to decide which ones are nav redesign variables and which ones are not.

Reply:
I used `navredesign` prefix since it was part of Nav Redesign project: pfsHM7-Ao-p2.
However, the naming was not ideal. The more context is available in this issue: https://github.com/Automattic/dotcom-forge/issues/7766 along with the related links. Does this make sense?

Reply:
So, basically to clarify which changes were introduced as part of Nav Redesign?

Reply:
> I can see why you'd want to stick to that convention, @chad1008, but when there's no difference between --color-sidebar-submenu-selected-hover-text and --color-navredesign-sidebar-submenu-selected-hover-text, that does add some confusion.

Oh, it's definitely confusing! 😅 

My rationale for using the `navredesign` variables is that those appear to be the ones that impact the `Hosting/My Home` view we're updating, while the "normal" variables without that prefix target the rest of Calypso, at least in practice for the items I've been working on.

This is part of what made this a more confusing process to update (and test, thanks again @nightnei!). The "Hosting/My Home" menu is basically a "Calypso in wp-admin" menu, because it's the one set of Calypso screens we still serve when a site owner has `wp-admin` set as their preferred interface. That separate menu, it turns out, has its own separate variables! yay!

I really like the issue @okmttdhr shared above. It would be great to standardize and simplify these variables across the different places they're being used.

Reply:
>My rationale for using the `navredesign` variables is that those appear to be the ones that impact the `Hosting/My Home` view we're updating, while the "normal" variables without that prefix target the rest of Calypso, at least in practice for the items I've been working on.

This makes sense to me!

>Because all the CSS vars in the L632-L655 block in client/my-sites/sidebar/style.scss use the --color-navredesign prefix, I can see why you'd want to stick to that convention, @chad1008, but when there's no difference between --color-sidebar-submenu-selected-hover-text and --color-navredesign-sidebar-submenu-selected-hover-text, that does add some confusion.

Something to consider is that if a value is used in a totally different UI, it will get more difficult to update it later. 
The idea behind having a new prefix is for safety reasons; Nav Unification and Nav Redesign have different ideas about the color schema.
I agree that these things might increase complexity, so I think it would be ideal to redesign the naming conventions, taking into account where they are used and so on.
As stated in the Issue: https://github.com/Automattic/dotcom-forge/issues/7766, I believe that many "prefixed variables" could be retired by introducing Core color schemes as possible to their original specifications.

Reply:
> My rationale for using the navredesign variables is that those appear to be the ones that impact the Hosting/My Home view we're updating

Thanks for the clarification 👍

For reference, https://github.com/Automattic/dotcom-forge/issues/7884 has more details on the `Hosting/My Home` definition.

I still don't fully understand why we need separate variables to style those elements, but it makes sense to me to follow the existing convention and potentially address this in the future.

Reply:
Hi all! Thanks @okmttdhr for the ping. Yeah, we add the new `navredesign` variables specifically for the "Hosting" Calypso pages. We did that because the rest of Calypso (nav-unification) pages don't seem to match Core for some reason, and we don't want to break them in the context of Untangling project.

What is the goal of the PRs? Are we just fixing the "Hosting" pages or also the rest of Calypso pages, to match with the wp-admin scheme? Or also "fixing" all Calypso pages?